src/main/java/io/cryostat/recordings/RecordingArchiveHelper.java

```diff
@@ -962,6 +986,182 @@ private long getFileSize(String recordingName) {
         }
     }
 
+    public boolean gzip(Path originalFile) {
```

Comment:
Does this need to be `public? In theory the ideal would be for this class to manage compressing recordings as they come in and we need to write them to disk, and anywhere that we read the files should transparently handle decompressing the stream, so this method can actually be a private implementation detail.

Reply:
Sorry, I forgot to change it to `private` after moving the `RecordingsPostHandler` methods to `RecordingArchiveHelper`. I will change it. :)